apps/src/music/views/MiniMusicPlayer.tsx

```diff
@@ -97,6 +98,27 @@ const MiniPlayerView: React.FunctionComponent<MiniPlayerViewProps> = ({
     setCurrentProjectId(undefined);
   }, []);
 
+  const getPackImageUrl = (packId: string) => {
```

Comment:
Would it be helpful to make this a function in `MusicLibrary` in case it's useful elsewhere?

Reply:
Great question.  I had talked myself out of needing it yet, but have now done it.